### DISCUSSION TITLE
Autolink and External Link

### DISCUSSION THREAD

**Discussion by cjmaxik** [thephpleague/commonmark#680] [Q&A] [answered]
Hello!
As I can tell, the External Link extension does not affect links created by the Autolink extension. Is that intended behavior?

**Comment by colinodell** [comment, marked as answer]
The External Link extension should act on **all** links, including auto links.  It seems you have found a bug!  I will create a new issue to get this resolved in 1.6.x.

**Reply by colinodell** [reply]
This has been fixed in [1.6.5](https://github.com/thephpleague/commonmark/releases/tag/1.6.5) (via #681).  Thanks for reporting this!
